assets/js/main.js

```diff
@@ -227,5 +271,61 @@ window.addEventListener('resize', () => {
     showDesktopElements()
 })
 
-// Logout listner
-document.querySelector('[js-header-auth-logout]').addEventListener('click', userLogout)
```

Comment:
We still need to allow logging out.

Reply:
Not sure what you mean? Why we need to allow logging out? Based on [this commit](https://github.com/codemagic-ci-cd/codemagic-docs/pull/841/commits/e43bc52c53d5b02a8a25cea569409681d1980e3e) these were your changes.